src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -67,6 +67,358 @@ public static ControlFlowGraph Create(IBlockOperation body)
             return new ControlFlowGraph(blocks.ToImmutableAndFree(), region);
         }
 
+        private static void MarkReachableBlocks(ArrayBuilder<BasicBlock> blocks)
+        {
```

Comment:
Nit: A general question: would it help debuggability to have the dictionaries, hash sets and array builders holder ordinals of blocks instead of block references?

Reply:
> A general question: would it help debuggability to have the dictionaries, hash sets and array builders holder ordinals of blocks instead of block references?

I don't think so. If one needs an ordinal, it is available from the block, it is much harder to get a block from its ordinal, while in debugger.

---
In reply to: [177189348](https://github.com/dotnet/roslyn/pull/25698#discussion_r177189348) [](ancestors = 177189348)